doc/compodoc_sources/how-to-guides/end-to-end-tests.md

```diff
@@ -94,7 +94,24 @@ option](https://argos-ci.com/docs/playwright#argosscreenshotpage-name-options)
 in `argosScreenshot()`. Set a default threshold with `argos upload
 --threshold=...`
 
-[aria-busy]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-busy
-[Playwright]: https://playwright.dev/
-[Playwright UI]: https://playwright.dev/docs/test-ui-mode
-[Trace Viewer]: https://playwright.dev/docs/trace-viewer
```

Comment:
aren't we losing any external links here now?

Reply:
Deleted this inadvertently. It’s restored now